### PR TITLE
qa/tests: advanced version to reflect the latest 16.2.6 release

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
https://pulpito.ceph.com/yuriw-2021-09-21_13:35:35-upgrade:pacific-p2p-pacific-distro-basic-smithi

**Review by jdurgin — APPROVED** [review]

**Comment by neha-ojha** [comment]
api test failure unrelated - https://tracker.ceph.com/issues/46032

**Comment by neha-ojha** [comment]
jenkins test api

**Comment by neha-ojha** [comment]
> https://pulpito.ceph.com/yuriw-2021-09-21_13:35:35-upgrade:pacific-p2p-pacific-distro-basic-smithi

Failures tracked in:

- https://tracker.ceph.com/issues/52590
- https://tracker.ceph.com/issues/51652

File: qa/suites/upgrade/pacific-p2p/pacific-p2p-stress-split/6-final-workload/rbd-python.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ meta:
    librbd python api tests
 tasks:
 - workunit:
-    tag: v16.2.4
+    tag: v16.2.5
```
Comment (neha-ojha) [inline qa/suites/upgrade/pacific-p2p/pacific-p2p-stress-split/6-final-workload/rbd-python.yaml:6]: should we install 16.2.6 here?

**Review by neha-ojha — APPROVED** [review]

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2021-09-21_23:01:35-upgrade:pacific-p2p-pacific-distro-basic-smithi/

**Comment by tchaikov** [comment]
jenkins test signed

**Pull request merged**